cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/common/DeploymentMetaProvider.java

```diff
@@ -1,56 +0,0 @@
-/*
```

Comment:
Removed these files specific to Deployment and StatefulSet. Took corresponding changes from https://github.com/cdapio/cdap/pull/12522